Test/Init.lean

```diff
@@ -0,0 +1,42 @@
+import cvc5
+
+
+
```

Comment:
This was for readability. I personally struggle to read code with no visual separation between parts of the code that are not related. Especially since we don't indent commands in lean. Removed the empty lines.